src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -105,20 +107,70 @@ public void storeMessage(Message message) {
     return messages;
   }
 
+  public static class User {
```

Comment:
Do you need this here? I'd prefer if you remove this class and instead import the one you define in user.java